### ISSUE TITLE
Doesn't run on Linux Mint

### ISSUE THREAD

**Issue by ayan2k26** [sedwards2009/extraterm#420]
I am running the latest version of Linux Mint with all programs updated as for writing. I have tried running the linux executable and installing the deb package, Getting the following error - 

```
2023-03-27 08:41:52.428 INFO [MainConfig] Looking for application_paths.json at '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/application_paths.json'
2023-03-27 08:41:52.430 INFO [ConfigDatabase #0] Reading user configuration from /home/ayan/.config/extratermqt/extraterm.json
2023-03-27 08:41:52.430 WARN [ConfigDatabase #0] Unable to find config for key  general
2023-03-27 08:41:52.431 WARN [ConfigDatabase #0] Unable to find config for key  command_line_action
2023-03-27 08:41:52.431 WARN [ConfigDatabase #0] Unable to find config for key  session
2023-03-27 08:41:52.432 INFO [main #0] Recording logs to  /home/ayan/.config/extratermqt/extraterm.log
2023-03-27 08:41:52.507 INFO [main #0] User extension directory is: /home/ayan/.config/extratermqt/extensions
2023-03-27 08:41:52.508 INFO [ExtensionManager #0] Scanning '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions' for extensions.
2023-03-27 08:41:52.510 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/About'.
2023-03-27 08:41:52.512 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/Autocomplete'.
2023-03-27 08:41:52.512 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/Autolink'.
2023-03-27 08:41:52.513 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/Colorizer'.
2023-03-27 08:41:52.513 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/CommunityTerminalThemes'.
2023-03-27 08:41:52.514 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/CopyBlock'.
2023-03-27 08:41:52.515 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/CopyLink'.
2023-03-27 08:41:52.515 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/DefaultKeybindings'.
2023-03-27 08:41:52.516 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/DefaultTerminalTheme'.
2023-03-27 08:41:52.516 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/DirectoryCommands'.
2023-03-27 08:41:52.517 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/DownloadBlock'.
2023-03-27 08:41:52.518 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/Find'.
2023-03-27 08:41:52.518 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/ITermColorsTerminalThemeProvider'.
2023-03-27 08:41:52.519 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/ImageBlock'.
2023-03-27 08:41:52.520 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/InjectShellIntegration'.
2023-03-27 08:41:52.521 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/InsertEmoji'.
2023-03-27 08:41:52.524 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/InternalCommands'.
2023-03-27 08:41:52.524 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/OpenLink'.
2023-03-27 08:41:52.524 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/StyleGuide'.
2023-03-27 08:41:52.525 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/TLDRPages'.
2023-03-27 08:41:52.526 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/TerminalTitle'.
2023-03-27 08:41:52.526 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/Tips'.
2023-03-27 08:41:52.527 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/UnixSessionBackend'.
2023-03-27 08:41:52.528 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/UnixSessionEditor'.
2023-03-27 08:41:52.528 INFO [ExtensionManager #0] Read extension metadata from '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/UpdateChecker'.
2023-03-27 08:41:52.529 INFO [ExtensionManager #0] Starting extension 'about'
2023-03-27 08:41:52.546 INFO [ExtensionManager #0] Starting extension 'autocomplete'
2023-03-27 08:41:52.548 INFO [ExtensionManager #0] Starting extension 'autolink'
2023-03-27 08:41:52.550 INFO [ExtensionManager #0] Starting extension 'colorizer'
2023-03-27 08:41:52.554 INFO [ExtensionManager #0] Starting extension 'community-terminal-themes'
2023-03-27 08:41:52.554 INFO [ExtensionManager #0] Starting extension 'copy-block'
2023-03-27 08:41:52.556 INFO [ExtensionManager #0] Starting extension 'copy-link'
2023-03-27 08:41:52.557 INFO [ExtensionManager #0] Starting extension 'default-keybindings'
2023-03-27 08:41:52.557 INFO [ExtensionManager #0] Starting extension 'default-terminal-theme'
2023-03-27 08:41:52.557 INFO [ExtensionManager #0] Starting extension 'directory-commands'
2023-03-27 08:41:52.558 INFO [ExtensionManager #0] Starting extension 'download-block'
2023-03-27 08:41:52.566 INFO [ExtensionManager #0] Starting extension 'find'
2023-03-27 08:41:52.569 INFO [ExtensionManager #0] Starting extension 'itermcolors-terminal-theme-provider'
2023-03-27 08:41:52.575 DEBUG [itermcolors-terminal-theme-provider] ITermColorTerminalThemeProvider activating
2023-03-27 08:41:52.576 INFO [ExtensionManager #0] Starting extension 'image-block'
2023-03-27 08:41:52.578 INFO [ExtensionManager #0] Starting extension 'inject-shell-integration'
2023-03-27 08:41:52.578 INFO [ExtensionManager #0] Starting extension 'insert-emoji'
2023-03-27 08:41:52.583 INFO [ExtensionManager #0] Starting extension 'internal-commands'
2023-03-27 08:41:52.583 INFO [ExtensionManager #0] Starting extension 'open-link'
2023-03-27 08:41:52.584 INFO [ExtensionManager #0] Starting extension 'styleguide'
2023-03-27 08:41:52.584 WARN [ExtensionManager #0] Unable to load /home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/StyleGuide/dist/StyleGuideExtension.cjs. Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/extensions/StyleGuide/dist/StyleGuideExtension.cjs' imported from /home/ayan/Downloads/extratermqt-0.71.0-linux-x64/main/dist/main.cjs
2023-03-27 08:41:52.584 INFO [ExtensionManager #0] Starting extension 'tldr-pages'
2023-03-27 08:41:52.586 INFO [ExtensionManager #0] Starting extension 'terminal-title'
2023-03-27 08:41:52.592 INFO [ExtensionManager #0] Starting extension 'tips'
2023-03-27 08:41:52.593 INFO [ExtensionManager #0] Starting extension 'unix-session-backend'
2023-03-27 08:41:52.609 INFO [ExtensionManager #0] Starting extension 'unix-session-editor'
2023-03-27 08:41:52.615 INFO [ExtensionManager #0] Starting extension 'update-checker'
2023-03-27 08:41:52.624 WARN [ConfigDatabase #0] Unable to find config for key  system
2023-03-27 08:41:52.663 INFO [LocalHttpServer #0] Local HTTP IPC server running on http://127.0.0.1:33241/79f293085a2a627310b89c3b9f6ad3c1

/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/main/dist/main.cjs:13541
          throw new Error("Unsupported OpenType signature " + signature);
                ^
Error: Unsupported OpenType signature ﾀￕ
    at parseBuffer (/home/ayan/Downloads/extratermqt-0.71.0-linux-x64/main/dist/main.cjs:13541:17)
    at /home/ayan/Downloads/extratermqt-0.71.0-linux-x64/main/dist/main.cjs:13708:22
    at /home/ayan/Downloads/extratermqt-0.71.0-linux-x64/main/dist/main.cjs:13441:11
    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3)
panic: Post "/command": unsupported protocol scheme ""

goroutine 1 [running]:
main.runCommand({0x0, 0x0}, 0xc000127f08)
	/home/runner/work/extraterm-launcher/extraterm-launcher/cmd/extraterm-launcher/extraterm-launcher.go:200 +0x2f4
main.runOpenWindowAtCommand({0x0, 0x0}, {0xc0000201c4, 0x31})
	/home/runner/work/extraterm-launcher/extraterm-launcher/cmd/extraterm-launcher/extraterm-launcher.go:135 +0xd9
main.main()
	/home/runner/work/extraterm-launcher/extraterm-launcher/cmd/extraterm-launcher/extraterm-launcher.go:40 +0x73

```

**Comment by sedwards2009** [comment]
It looks like a bad font. Do you know which font you selected last?

**Comment by ayan-iiitd** [comment]
@sedwards2009 yeah figured; since it seemed similar to https://github.com/sedwards2009/extraterm/issues/402

I installed a few otf fonts and changed the value in `extraterm.json` for got different error. 

![image](https://user-images.githubusercontent.com/59263847/228144401-4f6de93e-ac6a-4068-bd7e-180c409a7579.png)

Then I just changed the value to `"monospace"` which seems to work.

But I am noticing something, I am not getting the frame like structure in my terminal outputs, also the command `from` doesn't seem to work, it is prompting me to install something related to email. What exactly am I doing wrong here?

![image](https://user-images.githubusercontent.com/59263847/228145336-87df681a-b920-41d9-a13e-ed51b64f3e35.png)
